start_admin_sshd.sh

```diff
@@ -47,6 +47,26 @@ get_user_data_keys() {
     ( IFS=$'\n'; echo "${valid_keys[*]}" )
 }
 
+# Export proxy environment variables for all users' login shells
+# Match the values of the proxy environment variables given to the admin container
+set_up_proxy_env_vars() {
+  PROXY_ENV_VAR_SETUP_SCRIPT="/etc/profile.d/br-proxy-env.sh"
```

Comment:
Sorry I didn't notice this before, but could you set this to `local -r proxy_env_var_setup_script` or `declare -r PROXY_ENV_VAR_SETUP_SCRIPT` outside of the function and with the other constants?